javascript/mcp-server/src/index.ts

```diff
@@ -8,6 +8,71 @@ import { ForeverVM } from '@forevervm/sdk'
 import fs from 'fs'
 import path from 'path'
 import os from 'os'
+import { Command } from 'commander'
+
+// Install ForeverVM
+// For Claude Desktop, this function adds ForeverVM to the claude_desktop_config.json file
+async function installForeverVM(options: { claude: boolean }) {
+  let forevervmToken = getForeverVMToken()
+
+  if (!forevervmToken) {
+    console.error('ForeverVM token not found. Please set up ForeverVM first.')
+    throw new Error('ForeverVM token not found. Please set up ForeverVM first.')
```

Comment:
capitalization nit
```suggestion
    console.error('foreverVM token not found. Please set up foreverVM first.')
    throw new Error('foreverVM token not found. Please set up foreverVM first.')
```

Reply:
is forever always supposed to be lowercase?

Reply:
in copy, yes. in code, yes except for camelcase/pascal case (e.g. the javascript and python SDK classes are `ForeverVM` rather than `foreverVM` because it's idiomatic to start class names in those languages with a capital letter)

Reply:
this is consistent but it's worth discussing, I'm open to ForeverVM if we are consistent with it. Opened up on slack since it will be easier to follow: https://jamsocket.slack.com/archives/C02VDU8D9KN/p1739482561648019